app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+
+  def new
+    
+  end
+
+  def create
+    @user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
@ivanraybuglosa The `@user` variable is not being used in a template. Better to just use `user`